From b2deca683e0d57d9313c267d15c798f420ae3701 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=A0=D1=83=D1=81=D0=BB=D0=B0=D0=BD=20=D0=98=D0=B6=D0=B1?=
 =?UTF-8?q?=D1=83=D0=BB=D0=B0=D1=82=D0=BE=D0=B2?= <lrn1986@gmail.com>
Date: Mon, 13 Apr 2015 02:03:42 +0000
Subject: [PATCH 03/20] [W32] Use confgured Python interpreter

Configure script has tests for Python (AC_PATH_PROG(PYTHON, python)),
use the interpretor they provide (which is overridable by user by
the way of passing PYTHON=... to configure) instead of the first
thing found in PATH.

Otherwise wrong version of Python might be used. Mingw-python,
for example, which has commandline length limit, which results in
errors like:
/usr/bin/env: python: Argument list too long
GNUmakefile:82317: recipe for target
'DerivedSources/WebInspectorUI/GResourceBundle.xml' failed
---
 Source/WebInspectorUI/GNUmakefile.am | 2 +-
 Tools/gtk/GNUmakefile.am             | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Source/WebInspectorUI/GNUmakefile.am b/Source/WebInspectorUI/GNUmakefile.am
index 67b2477..25cdc41 100644
--- a/Source/WebInspectorUI/GNUmakefile.am
+++ b/Source/WebInspectorUI/GNUmakefile.am
@@ -13,7 +13,7 @@ ${GENSOURCES_WEBINSPECTOR_UI}/GResourceBundle.xml: GNUmakefile \
 	$(srcdir)/Tools/gtk/generate-inspector-gresource-manifest.py
 	$(AM_V_GEN)
 	$(AM_V_at)mkdir -p ${GENSOURCES_WEBINSPECTOR_UI}
-	$(AM_V_at)$(srcdir)/Tools/gtk/generate-inspector-gresource-manifest.py --output $@ $(inspector_files)
+	$(AM_V_at)$(PYTHON) $(srcdir)/Tools/gtk/generate-inspector-gresource-manifest.py --output $@ $(inspector_files)
 
 ${GENSOURCES_WEBINSPECTOR_UI}/GResourceBundle.c: DerivedSources/WebInspectorUI/GResourceBundle.xml
 	$(AM_V_GEN)
diff --git a/Tools/gtk/GNUmakefile.am b/Tools/gtk/GNUmakefile.am
index 757207a..61cff01 100644
--- a/Tools/gtk/GNUmakefile.am
+++ b/Tools/gtk/GNUmakefile.am
@@ -35,7 +35,7 @@ docs_build_dependencies += \
 	gtkdoc-webkitdom.cfg
 
 docs-build.stamp: $(docs_build_dependencies)
-	CC=$(CC) $(srcdir)/Tools/gtk/generate-gtkdoc
+	CC=$(CC) $(PYTHON) $(srcdir)/Tools/gtk/generate-gtkdoc
 	@touch docs-build.stamp
 
 clean-local: doc-clean-local
-- 
1.8.5.3

